src/main/java/uk/gov/pay/api/exception/mapper/CreateChargeConnectorErrorResponseExceptionMapper.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.pay.api.exception.mapper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.api.exception.CreateChargeConnectorErrorResponseException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static uk.gov.pay.api.model.PaymentError.Code.P0198;
+import static uk.gov.pay.api.model.PaymentError.Code.P0199;
+import static uk.gov.pay.api.model.PaymentError.aPaymentError;
+
+public class CreateChargeConnectorErrorResponseExceptionMapper implements ExceptionMapper<CreateChargeConnectorErrorResponseException> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CreateChargeConnectorErrorResponseException.class);
+
+    @Override
+    public Response toResponse(CreateChargeConnectorErrorResponseException exception) {
+
+        int errorStatus = exception.getErrorStatus();
+
+        if (errorStatus == NOT_FOUND.getStatusCode()) {
+            LOGGER.error("Authorization succeeded but Connector response was Gateway account not found: {}", exception.getErrorBody());
```

Comment:
There is a story in the backlog for logging.
And for error messages back to the user I'll mention it in stand up to see if they are interested on them